app/components/forms/session-speaker-form.js

```diff
@@ -302,6 +302,14 @@ export default Component.extend(FormMixin, {
     return groupBy(this.get('fields').toArray(), field => field.get('form'));
   }),
 
+  noSpeakerExists: computed('speakers', function() {
+    return (this.get('speakers').length === 1);
+  }),
+
+  noSessionExists: computed('sessions', function() {
+    return (this.get('sessions').length === 1);
```

Comment:
Why === 1 ?

Reply:
Add a comment above the computed property since this can be confusing.

Reply:
Okay I'll add a comment and also if someone finds a more appropriate approach using unloadRecord or something can replace this method till then I think we can merge this as for the events with no sessions the heading looks very absurd without the helper text.